### DISCUSSION TITLE
Why there are 2 different Build Phases in README.md

### DISCUSSION THREAD

**Discussion by mustafaozhan** [icerockdev/moko-resources#449] [Q&A]
Given that in README.md we have 

With `org.jetbrains.kotlin.native.cocoapods`
In Xcode add Build Phase (at end of list) with script:

```
"$SRCROOT/../gradlew" -p "$SRCROOT/../" :yourframeworkproject:copyFrameworkResourcesToApp \
    -Pmoko.resources.BUILT_PRODUCTS_DIR=$BUILT_PRODUCTS_DIR \
    -Pmoko.resources.CONTENTS_FOLDER_PATH=$CONTENTS_FOLDER_PATH\
    -Pkotlin.native.cocoapods.platform=$PLATFORM_NAME \
    -Pkotlin.native.cocoapods.archs="$ARCHS" \
    -Pkotlin.native.cocoapods.configuration=$CONFIGURATION 
```
Without `org.jetbrains.kotlin.native.cocoapods`
In Xcode add Build Phase (at end of list) with script:

```
"$SRCROOT/../gradlew" -p "$SRCROOT/../" :yourframeworkproject:copyFrameworkResourcesToApp \
    -Pmoko.resources.PLATFORM_NAME=$PLATFORM_NAME \
    -Pmoko.resources.CONFIGURATION=$CONFIGURATION \
    -Pmoko.resources.ARCHS="$ARCHS" \
    -Pmoko.resources.BUILT_PRODUCTS_DIR=$BUILT_PRODUCTS_DIR \
    -Pmoko.resources.CONTENTS_FOLDER_PATH=$CONTENTS_FOLDER_PATH 
```

In my project I have `org.jetbrains.kotlin.native.cocoapods` and actually both approach is working. Then I start thinking why we have the first option mentioned in README.md at all ? Is the first option better performing with `org.jetbrains.kotlin.native.cocoapods` ?

**Comment by Ustimov** [comment]
I suppose you have a dynamic framework. In this case you don't need any additional build phase.

**Reply by mustafaozhan** [reply]
Hey @Ustimov thank you for your answer, but no actually I have a static framework
